docker/settings/TOKEN.example

```diff
@@ -0,0 +1 @@
+notactuallyatokenyoushallnotpassblargh
```

Comment:
Why can't this go into `docker/.env`?

Reply:
becuase if we deploy an image, anything in env will show up in inspect iimage (including any tokens used for testing.